### ISSUE TITLE
[Feature Request][TL] Maybe need an annotation to disable part of the ast be translated into TL

### ISSUE THREAD

**Issue by LeiWang1999** [microsoft/BitBLAS#221]
Tile Language, or TensorIR Script, uses Python as its interface and automatically translates Python AST into TIR AST. However, this automatic translation can sometimes lead to inconvenient behavior.

For instance, consider the following example:

```python
sub_func = get_function(condition)

# Depending on the condition, sub_func requires a different number of arguments
if condition:
    sub_func requires three arguments
else:
    sub_func requires two arguments.

@T.prim_func
def function(
      A, B, C
):
   if condition is True:
      sub_func(A, B, C)
   else:
      sub_func(A, C)
```

The parser will transform this code into T.if_then_else(cond, sub_func(A, B, C), sub_func(A, C)). However, the parser must evaluate both branches of the if-then-else expression. This can cause issues when sub_func has a different number of required arguments based on the condition.

For example, if condition is True, sub_func expects three arguments (i.e., sub_func(A, B, C)). But because the parser also tries to evaluate the else branch (sub_func(A, C)), it throws an error like unexpected keyword argument, as sub_func is defined to take three arguments, not two. This leads to crashes since the parser is unable to reconcile the differing argument counts in both branches of the expression.

**Comment by LeiWang1999** [comment]
I think one solution may be implementing a `skip_translation` annotation to skip the translate that we can use python interface to 

```python
@T.prim_func
def function(
      A, B, C
):
   @T.skip_translation
   if condition is True:
      sub_func(A, B, C)
   else:
      sub_func(A, C)
```

**Comment by LeiWang1999** [comment]
or we could enhance tensor ir script, when the parser came across if statement, we should checkout whether the condition is an dedicated value, otherwise we should keep the statement to simplify the expression.

**Comment by LeiWang1999** [comment]
resolved with pr  #222
